src/controls/PrimitiveArrayControl.tsx

```diff
@@ -84,13 +85,7 @@ export function PrimitiveArrayControl({
   }
 
   return (
-    <Form.Item
-      id={id}
-      name={path}
-      label={label}
-      required={required}
-      {...formItemProps}
-    >
+    <Form.Item id={id} label={label} required={required} {...formItemProps}>
```

Comment:
name here was causing collisions with name in Form.List, so now Form.Item is basically just for visuals.